### PR TITLE
[Zoom] Create Zoom Event Collector

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4507)

## Description
Create new Zoom Logs Event Collector.
Fetching operation logs and sign In / sign out activity report

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.py [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+PRODUCT = 'Zoom'
+
+OAUTH_TOKEN_GENERATOR_URL = 'https://zoom.us/oauth/token'
+# The token’s time to live is 1 hour,
+# two minutes were subtract for extra safety.
+TOKEN_LIFE_TIME = timedelta(minutes=58)
+
+INVALID_CREDENTIALS = 'Invalid credentials. Verify that your credentials are valid.'
+INVALID_API_SECRET = 'Invalid API Secret. Verify that your API Secret is valid.'
+INVALID_ID_OR_SECRET = 'Invalid Client ID or Client Secret. Verify that your ID and Secret is valid.'
```
Comment (bziser) [inline Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.py:26]:
```suggestion
INVALID_ID_OR_SECRET = 'Invalid Client ID or Client Secret. Verify that your ID and Secret are valid.'
```

File: Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+            first_fetch_time: datetime, The first fetch time as configured in the integration params.
+            limit: int, the limit of the results to return. (is received only in zoom-get-events command)
+        Returns:
+            Tuple:
+                str: The time of the latest event fetched.
+                List: A list containing the events.
+        """
+
+        results: list[dict] = []
+        next_page_token: str = ''
```
Comment (bziser) [inline Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.py:150]: why needed `:str`?

File: Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.py [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+        next_last_time = last_time
+        first_page = True
+
+        demisto.debug(f"Last run before the fetch run: {last_time}")
+        start_date = first_fetch_time if not last_time else dateparser.parse(last_time).replace(tzinfo=timezone.utc)
+        end_date = datetime.now(timezone.utc) + timedelta(days=1)
+
+        demisto.debug(f"Start get logs from: {start_date} to: {end_date}")
```
Comment (bziser) [inline Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.py:158]:
```suggestion
        demisto.debug(f"Starting to get logs from: {start_date} to: {end_date}")
```

**Review by bziser — COMMENTED** [review]
looks good

File: Packs/Zoom/Integrations/ZoomEventCollector/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+This is the Zoom event collector integration for Cortex XSIAM.
+This integration was integrated and tested with version 2.0.0 of Zoom
+
+## Configure Zoom Event Collector on Cortex XSOAR
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/Integrations/ZoomEventCollector/README.md:4]:
```suggestion
## Configure Zoom Event Collector on Cortex XSIAM
```

File: Packs/Zoom/Integrations/ZoomEventCollector/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/Integrations/ZoomEventCollector/README.md:6]:
```suggestion
1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automations & Feed Integrations**.
```

File: Packs/Zoom/Integrations/ZoomEventCollector/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+2. Search for Zoom Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Server URL (e.g. 'https://api.zoom.us/v2/') | False |
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/Integrations/ZoomEventCollector/README.md:12]:
```suggestion
    | Server URL (e.g., 'https://api.zoom.us/v2/') | False |
```

File: Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.yml [hunk 7]
```diff
@@ -0,0 +1,80 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: ZoomEventCollector
+  version: -1
+configuration:
+- display: Server URL (e.g. 'https://api.zoom.us/v2/')
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/Integrations/ZoomEventCollector/ZoomEventCollector.yml:9]:
```suggestion
- display: Server URL (e.g., 'https://api.zoom.us/v2/')
```

**Comment by ShirleyDenkberg** [comment]
@bziser Doc review completed.

**Review by bziser — APPROVED** [review]

**Pull request merged**